relay-profiling/src/sample/mod.rs

```diff
@@ -0,0 +1,60 @@
+use serde::{Deserialize, Serialize};
+
+use crate::native_debug_image::NativeDebugImage;
+use relay_event_schema::protocol::Addr;
+
+pub mod v1;
+pub mod v2;
+
+#[derive(Debug, Serialize, Deserialize, Clone, Default, PartialEq, Eq)]
+pub enum Version {
+    #[default]
+    Unknown,
+    #[serde(rename = "1")]
+    V1,
+    #[serde(rename = "2")]
+    V2,
+}
+
+#[derive(Clone, Debug, Default, Serialize, Deserialize)]
+pub struct Frame {
```

Comment:
I know you just moved the files, but it would be nice if we took the opportunity and cleaned it up a bit:

- all fields should be public
- documentation on types and fields